.github/actions/Ubuntu-build/entrypoint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+
+if [ -z "$GITHUB_WORKSPACE" ]; then
+    echo "GITHUB_WORKSPACE is not set"
+    exit 1
+fi
+
+DEST_DIR=$GITHUB_WORKSPACE/rpms
+
+echo "Copying prebuilt packages for $IMAGEMAGICK_VERSION to $DEST_DIR"
+mkdir -p "$DEST_DIR"
+
+if [ -e /etc/debian_version ]; then
+   cp /root/*.deb "$DEST_DIR" 
+else
+   if [ -e /etc/redhat-release ]; then
+      cp /root/rpmbuild/RPMS/x86_64/*.rpm  "$DEST_DIR"
+   fi
+ fi
```

Comment:
Why does the script embed some logic based on rpm based distros if it's specific to Ubuntu?